gammapy/estimators/tests/test_excess_map.py

```diff
@@ -219,6 +231,19 @@ def test_significance_map_estimator_map_dataset_on_off_with_correlation(simple_d
 
     assert result_mod["flux"].unit == u.Unit("cm-2s-1")
     assert_allclose(result_mod["flux"].data[0, 10, 10], 1.486806e-08, rtol=1e-3)
+    assert_allclose(result_mod["flux"].data.sum(), 5.254442192077636e-06, rtol=1e-8)
+
+    estimator_mod = ExcessMapEstimator(
+        0.11 * u.deg,
+        apply_mask_fit=False,
+        correlate_off=True,
+        spectral_model=PowerLawSpectralModel(index=10),
+    )
+    result_mod = estimator_mod.run(simple_dataset_on_off)
+
+    assert result_mod["flux"].unit == u.Unit("cm-2s-1")
+    assert_allclose(result_mod["flux"].data.sum(), 5.254442192077636e-06, rtol=1e-8)
+    # TODO: find a test where we can actually see a difference with the default case
```

Comment:
I think this would be important. For adding an assert on dnde should be fine? This should be different for both cases...